roles/scaffold_rm_facts/templates/module_utils/network_os/config/resource/resource.py.j2

```diff
@@ -10,13 +10,21 @@ is compared to the provided configuration (as dict) and the command set
 necessary to bring the current configuration to it's desired end-state is
 created
 """
-from ansible.module_utils.network.common.cfg.base import ConfigBase
-from ansible.module_utils.network.common.utils import to_list
+from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.cfg.base import (
```

Comment:
You can add a conditional check if the structure is collection import from the collection or else import from ansible to maintain backwards compatibility. Applicable at other places as well.

Reply:
Okay.